dist/releases/4.14/config.toml

```diff
@@ -0,0 +1,79 @@
+[[payload.ose-network-interface-bond-cni-container.ignore]]
+error = "ErrLibcryptoSoMissing"
+files = [
+  "/bondcni/bond",
+  "/bondcni/rhel9/bond",
+]
+
+[[payload.ose-network-tools-container.ignore]]
+error = "ErrLibcryptoSoMissing"
+files = [ "/usr/bin/ovnkube-trace" ]
+
+[[rpm.cri-o.ignore]]
+error = "ErrGoMissingTag"
+files = [
+  "/usr/bin/crio-status",
+  "/usr/bin/crio",
+]
+
+[[rpm.cri-o.ignore]]
+error = "ErrNotDynLinked"
+files = [ "/usr/bin/pinns" ]
+
+[[rpm.skopeo.ignore]]
+error = "ErrGoMissingTag"
+files = [ "/usr/bin/skopeo" ]
+
+[[payload.ose-machine-config-operator-container.ignore]]
+error = "ErrLibcryptoSoMissing"
+files = [ "/usr/bin/machine-config-daemon.rhel9" ]
+
+[[rpm.containernetworking-plugins.ignore]]
+error = "ErrGoMissingTag"
+files = [ "/usr/libexec/cni/dummy" ]
+
```

Comment:
Looks like you've copied it from 4.13 config, which I forgot to fix. Please see https://github.com/openshift/check-payload/pull/106